frontend/src/components/accounts/__tests__/AccountDetails.test.tsx

```diff
@@ -20,12 +20,6 @@ describe("<AccountDetails />", () => {
             storageUsage: "876",
             lockupAccountId: "gjturigjgkjnfidsjffjsa.lockup.near",
           }}
-          currentNearNetwork={{
```

Comment:
We now have network context in every test - via `renderI18nElement` (which should be probably renamed to `renderTestElement`)